compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -363,6 +364,16 @@ impl<'ll, 'tcx> IntrinsicCallMethods<'tcx> for Builder<'_, 'll, 'tcx> {
                 return;
             }
 
+            sym::vtable_size | sym::vtable_align => {
+                let vtable = args[0].immediate();
+                let idx = match name {
+                    sym::vtable_size => ty::COMMON_VTABLE_ENTRIES_SIZE,
+                    sym::vtable_align => ty::COMMON_VTABLE_ENTRIES_ALIGN,
+                    _ => bug!(),
+                };
+                meth::VirtualIndex::from_index(idx).get_usize(self, vtable)
```

Comment:
It would be nice to provide the same range metadata here as we do elsewhere: https://github.com/rust-lang/rust/blob/389352c4bbd56024eefd1566e06dcba04d223f92/compiler/rustc_codegen_ssa/src/glue.rs#L27-L33
i.e.
```suggestion
                let value = meth::VirtualIndex::from_index(idx).get_usize(self, vtable);
                if name == sym::vtable_align {
                    // Alignment is always nonzero.
                    self.range_metadata(value, WrappingRange { start: 1, end: !0 });
                }
                value
```